code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -529,4 +529,38 @@
 				emote("yaps happily")
 		else
 			if(M && stat != DEAD) // Same check here, even though emote checks it as well (poor form to check it only in the help case)
-				emote("growls")
\ No newline at end of file
+				emote("growls")
+
+
+//Sasha isn't even a corgi you dummy!
+/mob/living/simple_animal/corgi/Sasha
+	name = "Sasha"
+	real_name = "Sasha"
+	gender = FEMALE
+	desc = "It's a doberman, how intimidating!"
+	icon_state = "doby"
+	icon_living = "doby"
+	icon_dead = "doby_dead"
+	response_help  = "pets"
+	response_disarm = "bops"
+	response_harm   = "kicks"
+	var/turns_since_scan = 0

```

Comment:
It's your code how do you not know? Also if its never used then yes get rid of it